Description: Move libraries after files that need them to fix FTBFS with ld --as-needed.
Author: Logan Rosen <logan@ubuntu.com>

--- a/tests/Makefile
+++ b/tests/Makefile
@@ -64,8 +64,8 @@
 #    direct jumps, which leaves caller information out of the stack frame and
 #    makes functions not appear in the backtrace. We disable it.
 test-enable_stac%: test-enable_stac%.c build-flags
-	$(NICE_CC) $(ALL_CFLAGS) -lfiu -lpthread \
-		-rdynamic -fno-optimize-sibling-calls $< -o $@
+	$(NICE_CC) $(ALL_CFLAGS) \
+		-rdynamic -fno-optimize-sibling-calls $< -lfiu -lpthread -o $@
 
 c-run-%: %
 	$(NICE_RUN) ./$<
